.jenkinsci/debug-build.groovy

```diff
@@ -20,8 +21,11 @@ def doDebugBuild(coverageEnabled=false) {
     + " -e POSTGRES_PASSWORD=${env.IROHA_POSTGRES_PASSWORD}"
     + " --name ${env.IROHA_POSTGRES_HOST}"
     + " --network=${env.IROHA_NETWORK}")
-
-  def iC = dPullOrBuild.dockerPullOrUpdate()
+  def iC = dPullOrBuild.dockerPullOrUpdate("${platform}-develop",
+                                           "https://raw.githubusercontent.com/hyperledger/iroha/${env.GIT_COMMIT}/docker/develop/${platform}/Dockerfile",
```

Comment:
Maybe it's possible to use some global var for that?

Reply:
`iC` is actually "not-so-global". It is used in Linux builds, but not required for MacOS build. `dPullOrBuild.dockerPullOrUpdate()` returns Groovy object, although global variable can only be defined in `environment` section of Jenkinsfile and only of type `string`

Reply:
I mean `https://raw.githubusercontent.com/hyperledger/iroha/` substring